vue-components/stories/Checkbox.stories.ts

```diff
@@ -0,0 +1,25 @@
+import Checkbox from '@/components/Checkbox';
+import { Component } from 'vue';
+
+export default {
+	component: Checkbox,
+	// the `/` prefix in the title is needed for "Input" to appear as a folded navigation item, and not a headline
+	title: 'Checkbox',
+};
+
+export function all(): Component {
+	return {
+		components: { Checkbox },
+		template: `
+			<div>
+				<div><Checkbox label="Checked" :value="true" /></div>
```

Comment:
```suggestion
				<div><Checkbox label="Checked" aria-checked="true" :value="true" /></div>
```

Reply:
What about keeping up with the display of good practices? Would the aria-checked attribute be added here, in this order?

Reply:
No, since we are using a native checkbox in our template, which should still be accessible to a screen-reader.

Reply:
Similarly, we are not using `aria-disabled`. Most of these rules are intended to be used on completely custom build elements (i.e. most of the time `<span>` and `<div>`).

Reply:
thanks for the clarification!